release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -79,6 +79,17 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
 
 For more information, see xref:../installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[Installing a cluster on IBM Cloud in a restricted network].
 
+[id="ocp-4-15-installation-and-update-aws-wavelength-zones"]
+==== Installing a cluster in AWS Wavelength Zones
```

Comment:
We may need to add a more appropriate title as we are not installing a cluster, but extending nodes to WL.
Suggestions:
- Installing a cluster with support of AWS Wavelength Zones
- Installing a cluster on AWS extending nodes to Wavelength Zones